include/dlaf/eigensolver/tridiag_solver.h

```diff
@@ -70,19 +77,26 @@ void tridiagSolver(Matrix<BaseType<T>, Device::CPU>& tridiag, Matrix<BaseType<T>
 ///                of the second column is not used.
 /// @param evals [out] (n x 1) local matrix holding the eigenvalues of the the symmetric tridiagonal
 ///              matrix
-/// @param evecs [out] (n x n) distributed matrix holding the eigenvectors of the the symmetric tridiagonal
+/// @param evecs [out] (n x n) distributed matrix holding the eigenvectors of the the symmetric
```

Comment:
```suggestion
/// @param evecs [out] (n x n) distributed matrix holding the eigenvectors of the symmetric
```

Reply:
I'll do this separately. There were at least a few other doc-formatting changes that I noticed were off while doing this.

Reply:
#944.